ext/pdo/php_pdo.h

```diff
@@ -22,6 +22,9 @@
 extern zend_module_entry pdo_module_entry;
 #define phpext_pdo_ptr &pdo_module_entry
 
+ZEND_API zend_class_entry *pdo_dbh_ce;
+ZEND_API zend_object *pdo_dbh_new(zend_class_entry *ce);
```

Comment:
Maybe?
```suggestion
extern ZEND_API zend_class_entry *pdo_dbh_ce;
extern ZEND_API zend_object *pdo_dbh_new(zend_class_entry *ce);
```

Reply:
That didn't work before....trying it again though.